packages/ui/src/molecules/Gift/Gift.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+import type { ReactNode } from 'react'
+
+import Icon from '../../atoms/Icon'
+export interface GiftProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * A React component that will be rendered as an icon.
+   */
+  icon?: ReactNode
+  /**
+   * Label to be required for accessibility.
```

Comment:
Just to be more clear:
```suggestion
   * Label to be required for accessibility for the Icon.
```